microprofile-opentracing/README.adoc

```diff
@@ -11,6 +11,8 @@ The `microprofile-opentracing` quickstart demonstrates the use of the MicroProfi
 :standalone-server-type: default
 :archiveType: war
 :archiveName: {artifactId}
+:jbossHomeName: JBOSS_HOME
```

Comment:
no need to define this attribute here, already defined by the include of shared-docs/attributes